src/Middleware/HttpsPolicy/test/HstsMiddlewareTests.cs

```diff
@@ -15,13 +15,46 @@
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Testing;
+using Microsoft.Extensions.Options;
 using Microsoft.Net.Http.Headers;
 using Xunit;
 
 namespace Microsoft.AspNetCore.HttpsPolicy.Tests
 {
     public class HstsMiddlewareTests
     {
+        [Fact]
+        public void Ctor_ArgumentNextIsNull_ThrowsArgumentNullException()
+        {
+            Assert.Throws<ArgumentNullException>(() =>
+            {
+                _ = new HstsMiddleware(next: null, options: new OptionsWrapper<HstsOptions>(new HstsOptions()));
+            });
+        }
+
+        [Fact]
+        public void Ctor_ArgumentOptionsIsNull_ThrowsArgumentNullException()
+        {
+            Assert.Throws<ArgumentNullException>(() =>
+            {
+                _ = new HstsMiddleware(innerHttpContext => Task.CompletedTask, options: null);
+            });
+        }
+
+        [Fact]
+        public async Task Invoke_SetsHstsHeaderWithDefaultThirtyDaysMaxAge()
```

Comment:
How is this test different from the following SetOptionsWithDefault_SetsMaxAgeToCorrectValue? The only difference I can see is the port.

Reply:
Shall I remove it?

Reply:
Yes if it's not providing unique coverage

Reply:
Removed

Reply:
Thanks